src/Modules/Admin/Commands.Admin/help/Get-PowerBIActivityEvents.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version: https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.admin/get-powerbiactivityevents?view=powerbi-ps
+schema: 2.0.0
+---
+
+# Get-PowerBIActivityEvents
+
+## SYNOPSIS
+Retrieves the audit activity events for a Power BI tenant.
+
+## SYNTAX
+
+```
+Get-PowerBIActivityEvents -StartDateTime <String> -EndDateTime <String> [-ActivityType <String>]
+ [-ResultType <OutputType>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Retrieves the audit activity events for the calling user's tenant.
+Before you run this command, make sure you log in using Connect-PowerBIServiceAccount.
+This cmdlet requires the calling user to be a tenant administrator of the Power BI service.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-PowerBIActivityEvents -StartDateTime 2019-08-10T14:35:20 -EndDateTime 2019-08-10T18:25:50
+```
+
+### Example 2
+```powershell
+PS C:\> Get-PowerBIActivityEvents -StartDateTime 2019-08-10T14:35:20 -EndDateTime 2019-08-10T18:25:50 -ActivityType viewreport -ResultType 1
+```
+
+## PARAMETERS
+
+### -ActivityType
+Filters the activity records based on this activity type.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EndDateTime
+Specifies the end of a timespan to retrieve audit activity events. It should be in UTC format and ISO 8601 compliant. Both StartDateTime and EndDateTime should be within the same UTC day.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResultType
+Indicates the type of result that is returned by the cmdlet. Currently the supported values are:
+
+* JsonString: Result type is a string representation of list of Json objects.
```

Comment:
Remove, don't do bullets. Just keep "Specifies the type of the result that is returned by the cmdlet."
I think you have to do a rebuild to get this file to show the acceptable values